src/main/java/org/folio/consortia/domain/dto/SourceValues.java

```diff
@@ -9,8 +9,8 @@ public enum SourceValues {
   CONSORTIUM_FOLIO_INSTANCE("CONSORTIUM-FOLIO"),
   CONSORTIUM_MARC_INSTANCE("CONSORTIUM-MARC"),
   CONSORTIUM_LINKED_DATA_INSTANCE("CONSORTIUM-LINKED_DATA"),
-  CONSORTIUM("consortium"),
-  USER("user");
+  CONSORTIUM("CONSORTIUM"),
+  USER("USER");
```

Comment:
for settings endpoints we should keep lower case because it stored in jsonb in lower case.
And for sharing policies and roles we need upper case, because mod-roles-keycloak works with upper case for all enum values, so probably new enum is needed here